src/components/feedback/Snackbar/styles.less

```diff
@@ -0,0 +1,89 @@
+@import '~styles/mixins.less';
+@import '~styles/variables.less';
+
+.container {
+    --width-multiplier: 75px;
+    background-color: var(--color-black-alt1);
+    height: .grid-unit(6px)[];
+    width: calc(var(--width-multiplier)* var(--grid-unit));
+    border-radius: 4px;
+    display: flex;
+    align-items: center;
+    position: fixed;
+    justify-content: space-between;
+    transition: opacity .2s;
```

Comment:
Have you discussed the transition effect with Lotte?

Reply:
Yes